docs/convert.asciidoc

```diff
@@ -0,0 +1,63 @@
+[[convert-to-ecs]]
+== Converting an implementation to ECS
+
+A common schema helps you correlate and use data from various sources. 
+
+Fields for most Elastic modules and solutions (version 7.1 and later) are mapped
+to the Elastic Common Schema. You may want to map data from other
+implementations to ECS to help you correlate data across all of your products
+and solutions.
+
+Before you start a conversion, be sure that you understand the basic <link here>.
+
+[float]
+[[core-or-ext]]
+=== Core and extended fields
+
+* *Core fields.* Fields that are most common across all use cases are called *core fields*. 
++
+These generalized fields are used by analysis content
+(searches, visualizations, dashboards, alerts, machine learning jobs, reports)
+across use cases. Analysis content designed to operate on these
+fields should work properly on data from any relevant source. 
++
+Focus on populating these fields first. 
+
+* *Extended fields.* Any field that is not a core field is called an *extended field*. 
+Extended fields may apply to more narrow use cases, or may be more open
+to interpretation depending on the use case. Extended fields are more likely to
+change over time.
+
+Each {ecs} <<ecs-fields,field>> in a table is identified as core or extended.
+
+[float]
+[[ecs-comv]]
+=== An approach to mapping an existing implementation
+
+Here's the recommended approach for converting an existing implementation to {ecs}.
+
+. Start with Core fields.
++
+Populate core fields first. Look at your set of event fields, and find
+the appropriate ECS field name for each one. 
+
+. Move on to Extended fields.
++
+Map fields that may be specific to various data sources using {ecs} extended
+fields. Look at {ecs} extended fields, and decide how to populate these fields
+with the data you have available. Even if you have already mapped a field to an
+{ecs} core field, you can still map it to an extended field. 
+
+Populating both core and extended fields helps ensure reusability of ECS analysis
+content. 
+
+[float]
+[[ecs-comp]]
+=== ECS compatible or compliant?
+
+An implementation can be considered *compatible* with ECS as long it maps all
+existing fields to ECS fields.
+
+An implementation would be considered *compliant* with ECS only if it also
+populates as many of the ECS core fields as is practical. Duplication, adding
+metadata, or deriving values via manipulation and math are acceptable.
```

Comment:
In line with my other comment on compatible/compliant, we should consider removing this section.